google/cloud/networkconnectivity/quickstart/quickstart.cc

```diff
@@ -0,0 +1,39 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "google/cloud/networkconnectivity/hub_client.h"
+#include <iostream>
+#include <stdexcept>
+
+int main(int argc, char* argv[]) try {
+  if (argc != 2) {
+    std::cerr << "Usage: " << argv[0] << " project-id\n";
+    return 1;
+  }
+
+  namespace networkconnectivity = ::google::cloud::networkconnectivity;
+  auto client = networkconnectivity::HubServiceClient(
+      networkconnectivity::MakeHubServiceConnection());
+
+  auto const parent = std::string{"projects/"} + argv[1] + "/locations/global";
```

Comment:
I feel I should be asking something clever about location enabled services.  But I do not know enough to ask a smart question... So this will have to do: should we use the location stuff for this service?

Reply:
I did not check before you asked, but [the docs](https://cloud.google.com/network-connectivity/docs/reference/networkconnectivity/rest#service-endpoint) say there is only one endpoint. So no, we don't need to do location stuff for it.

Reply:
SGTM.